specification/schemas/Payment.json

```diff
@@ -0,0 +1,85 @@
+{
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "#/components/schemas/MinimalPayment"
+    },
+    {
+      "properties": {
+        "attributes": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "amount": {
+              "$ref": "#/components/schemas/PriceAmount"
+            },
+            "currency": {
+              "$ref": "#/components/schemas/Currency"
+            },
+            "paid_at": {
+              "oneOf": [
+                {
+                  "$ref": "#/components/schemas/Timestamp"
+                },
+                {
+                  "type": "null"
+                }
+              ]
+            },
+            "provider_response": {
+              "readOnly": true,
+              "type": [
+                "string",
+                "null"
+              ],
+              "example": "succeeded"
+            },
+            "created_at": {
+              "$ref": "#/components/schemas/Timestamp"
+            },
+            "updated_at": {
+              "$ref": "#/components/schemas/Timestamp"
+            }
+          }
+        },
+        "relationships": {
+          "type": "object",
+          "required": [
+            "organization",
+            "status"
```

Comment:
These should not be required in the base model, as we want them not to be required in a patch.